### PR TITLE
Add verify-packages-install script to this repo

### PR DESCRIPTION
**Pull request by sladyn98**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
This PR  verfies the installation of debs and rpms by moving the [verify stable packages](https://github.com/kubernetes/kubeadm/blob/ead9b04ef8c8001d1b673dc1a3b04eb0b708bb58/tests/e2e/packages/verify_packages_install_deb.sh#L80-L125) script to this repo.

#### Which issue(s) this PR fixes:
https://github.com/kubernetes/release/issues/821

#### Special notes for your reviewer:

This would need a test-infra job to monitor the verification of packages

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: hack/packages/verify-packages-install-deb.sh [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+  apt-get install -y "${PACKAGES_TO_TEST_EXT[@]}" "${PACKAGES_TO_TEST_K8S_VER[@]}"
+
+  echo "* testing binaries"
+
+  kubeadm version -o=short
+  kubectl version --client=true --short=true
+  kubelet --version
+  crictl --version
+  ls /opt/cni/bin/loopback
```
Comment (sladyn98) [inline hack/packages/verify-packages-install-deb.sh:119]: @xmudrii This is the part of the test missing in the existing script

**Comment by cpanato** [comment]
@sladyn98 can you take a look in the fail builds? maybe update the PR fix it

File: hack/packages/verify-packages-install-deb.sh [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+  echo "* removing packages"
+
+  apt-get remove -y --purge "${PACKAGES_TO_TEST[@]}"
+  apt-get autoremove -y
+done
```
Comment (cpanato) [inline hack/packages/verify-packages-install-deb.sh:125]: nit: new line

**Comment by xmudrii** [comment]
@cpanato As described in the issue (https://github.com/kubernetes/release/issues/821#issuecomment-829197754), it's currently not possible to proceed with this PR. The reason is that this approach doesn't work anymore, and we're still not using the `kubepkg`. I propose that we close this PR for now.

**Comment by saschagrunert** [comment]
@sladyn98 @cpanato do we have any update on this? Are we still working on it?

**Comment by cpanato** [comment]
> @sladyn98 @cpanato do we have any update on this? Are we still working on it?

will wait @sladyn98 reply, if not I think I can rebase and finish the work

**Pull request closed without merging**